handsontable/src/plugins/autofill/autofill.js

```diff
@@ -426,21 +426,22 @@ export class Autofill extends BasePlugin {
   getIndexOfLastAdjacentFilledInRow(cornersOfSelectedCells) {
     const data = this.hot.getData();
     const nrOfTableRows = this.hot.countRows();
+    const isEmptyCell = cellContent => cellContent === null || cellContent === void 0 || cellContent === '';
```

Comment:
```suggestion
    const isEmptyCell = cellContent => cellContent === null || cellContent === undefined || cellContent === '';
```

Reply:
Or better by using [that generic helper](https://github.com/handsontable/handsontable/blob/4832dd2386432c389dc2b2d7c7d275a126066932/handsontable/src/helpers/mixed.js#L59).